Document/0x05c-Reverse-Engineering-and-Tampering.md

```diff
@@ -1014,6 +1014,34 @@ $ frida-trace -p 1372 -a "libjpeg.so!0x4793c"
 
 To learn more about all options for advanced usage, check the [documentation in the official Frida website](https://frida.re/docs/frida-trace/ "documentation").
 
+##### JNI Tracing
+As detailed in the section on [Static Analysis](#static-analysis), the first argument passed to every JNI function is a JNI interface pointer. This pointer contains a table of functions that allows native code to access the Android Runtime. Spotting this pattern can help with understanding the type of functionality a library is performing, such as where it creates new strings or calls Java methods.
```

Comment:
I'd rather refer to the subsection. If not it makes it difficult to check.

```suggestion
As detailed in section [Reviewing Disassembled Native Code](#reviewing-disassembled-native-code), the first argument passed to every JNI function is a JNI interface pointer. This pointer contains a table of functions that allows native code to access the Android Runtime. Spotting this pattern can help with understanding the type of functionality a library is performing, such as where it creates new strings or calls Java methods.
```

Reply:
I'm not sure if i understand what's meant by "Spotting this pattern". Could you please tell what is it meant here?

Reply:
I was trying to explaining that seeing those JNI method calls can be very useful in understanding what a native library is doing. I have attempted to rewrite that in the latest commit, let me know if you think it is any better.